streamnative/README.md

```diff
@@ -18,8 +18,9 @@ The StreamNative integration collects the following types of [metrics][2]:
 
 1. Log into [StreamNative Cloud Console Account][3].
 2. Click the profile icon and navigate to the **Accounts & Accesses** tab.
-3. Find the Service Account with **Admin** permissions set to **Enabled**.
+3. Find the Service Account with **Admin** permissions or binding with `metrics-viewer` role.
    - If no Service Account exists, select **New -> Service Account** to create one, and make sure to enable the **Super Admin** option.
+   - To bind a service account with `metrics-viewer`, your refer the [metrics-viewer rolebinding](https://docs.streamnative.io/docs/cloud-metrics-api#metrics-authorization).
```

Comment:
```suggestion
   - To bind a service account with the `metrics-viewer` role, refer to the [metrics-viewer rolebinding][5] documentation. 
```
Links have to go in this numbering format and then at the bottom of the README, please add:
[5]: https://docs.streamnative.io/docs/cloud-metrics-api#metrics-authorization

Reply:
okay, sure let me update it.

Reply:
@aliciascott i updated based on your suggestion, it's ready for re-review. thx a lot.